packages/react-i18n/src/utilities/translate.tsx

```diff
@@ -19,14 +19,14 @@ const SEPARATOR = '.';
 
 const numberFormats = new Map<string, Intl.NumberFormat>();
 export function memoizedNumberFormatter(
-  locale: string,
-  options: Intl.NumberFormatOptions = {},
+  locales?: string | string[],
+  options?: Intl.NumberFormatOptions,
 ) {
-  const key = numberFormatCacheKey(locale, options);
+  const key = numberFormatCacheKey(locales, options);
   if (numberFormats.has(key)) {
     return numberFormats.get(key)!;
   }
-  const i = new Intl.NumberFormat(locale, options);
+  const i = new Intl.NumberFormat(locales, options);
```

Comment:
I guess passing `undefined` to `NumberFormat` is the same as passing an empty object?